src/gpodder/gtkui/main.py

```diff
@@ -1028,6 +1041,8 @@ def on_key_press(treeview, event):
                     self._search_episodes.hide_search()
                 else:
                     self.shownotes_object.hide_pane()
+            elif event.keyval == Gdk.KEY_Menu:
+                self.treeview_available_show_context_menu(self.treeChannels)
```

Comment:
Wrong treeview, should be treeAvailable.

Reply:
There was another mistake if this type there too. I refactored the code to make them impossible.